src/include/optimizer/operators.h

```diff
@@ -141,39 +151,38 @@ class LogicalLimit : public OperatorNode<LogicalLimit> {
 //===--------------------------------------------------------------------===//
 // Insert
 //===--------------------------------------------------------------------===//
-class LogicalInsert: public OperatorNode<LogicalInsert> {
+class LogicalInsert : public OperatorNode<LogicalInsert> {
  public:
   static Operator make(
-      storage::DataTable* target_table,
-      std::vector<char*>* columns,
-      std::vector<std::vector<peloton::expression::AbstractExpression*>*>* values);
+      storage::DataTable *target_table, std::vector<char *> *columns,
+      std::vector<std::vector<peloton::expression::AbstractExpression *> *> *
+          values);
 
-  storage::DataTable* target_table;
-  std::vector<char*>* columns;
-  std::vector<std::vector<peloton::expression::AbstractExpression*>*>* values;
+  storage::DataTable *target_table;
+  std::vector<char *> *columns;
+  std::vector<std::vector<peloton::expression::AbstractExpression *> *> *values;
 };
 
 //===--------------------------------------------------------------------===//
 // Delete
 //===--------------------------------------------------------------------===//
-class LogicalDelete: public OperatorNode<LogicalDelete> {
+class LogicalDelete : public OperatorNode<LogicalDelete> {
  public:
-  static Operator make(storage::DataTable* target_table);
+  static Operator make(storage::DataTable *target_table);
 
-  storage::DataTable* target_table;
+  storage::DataTable *target_table;
 };
 
 //===--------------------------------------------------------------------===//
 // Update
 //===--------------------------------------------------------------------===//
-class LogicalUpdate: public OperatorNode<LogicalUpdate> {
+class LogicalUpdate : public OperatorNode<LogicalUpdate> {
  public:
-  static Operator make(const parser::UpdateStatement* update_stmt);
+  static Operator make(const parser::UpdateStatement *update_stmt);
 
-  const parser::UpdateStatement* update_stmt;
+  const parser::UpdateStatement *update_stmt;
```

Comment:
Because the planner accepts a pointer to the parse tree as an argument for the update_plan constructor. I will refactor the update_plan and the optimizer.